assets/js/blocks/cart-checkout-shared/default-notice/editor.scss

```diff
@@ -0,0 +1,11 @@
+.wc-block-cart__page-notice {
+	margin: 0;
+	padding-right: 16px;
+	.components-notice__dismiss {
+		min-width: 24px;
+	}
+	svg {
+		width: 16px;
+		height: 16px;
+	}
+}
```

Comment:
Currently, those styles are meant for the default cart notice (that's fine), however, can we extend them to cover the compatibility notice as well?

The goal was to match `x` sizes here
<img width="282" alt="image" src="https://user-images.githubusercontent.com/6165348/183944134-dd1509a6-1001-49bb-b3d0-d4d6cd02efe8.png">